### PR TITLE
Add age, county, jurisdiction, race, sex, and county to tracked metrics

### PR DESCRIPTION
**Pull request by georgehelman**

### CODE REVIEW

**Comment by georgehelman** [comment]
https://github.com/deardurham/dear-petition/issues/265

File: dear_petition/petition/models.py [hunk 1]
```diff
@@ -332,3 +346,12 @@ class GeneratedPetition(TimeStampedModel):
     form_type = models.CharField(choices=FORM_TYPES, max_length=255)
     number_of_charges = models.IntegerField()
     batch_id = models.PositiveIntegerField()
```
Comment (copelco) [inline dear_petition/petition/models.py:348]: I'd suggest making all of the new fields read-only in the admin, so these values can't be modified.

**Review by robert-w-gries — CHANGES_REQUESTED** [review]
Is there any testing for the statistics yet? I think that could be a good addition if it won't take too much time. Otherwise, we can split it out to a new ticket

**Review by robert-w-gries — APPROVED** [review]

**Comment by georgehelman** [comment]
I'll split it into a new issue.

**Comment by georgehelman** [comment]
https://github.com/deardurham/dear-petition/issues/280

**Pull request merged**